front/nuxt.config.ts

```diff
@@ -59,86 +58,46 @@ export default defineNuxtConfig({
             charset: 'utf-8',
-            viewport: 'width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover',
+            viewport:
+                'width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover'
         }
     },
-    // pwa: {
-    //     scope: '/',
-    //     base: '/',
-    //     injectRegister: 'auto',
-    //     registerType: 'autoUpdate',
-    //     manifest: {
-    //         name: appName,
-    //         short_name: appName,
-    //         description: appDescription,
-    //         theme_color: "#1867c0",
-    //         background_color: "#1867c0",
-    //         icons: [
-    //             {
-    //                 src: 'pwa-64x64.png',
-    //                 sizes: '64x64',
-    //                 type: 'image/png'
-    //             },
-    //             {
-    //                 src: 'pwa-192x192.png',
-    //                 sizes: '192x192',
-    //                 type: 'image/png'
-    //             },
-    //             {
-    //                 src: 'pwa-512x512.png',
-    //                 sizes: '512x512',
-    //                 type: 'image/png',
-    //                 purpose: 'any'
-    //             },
-    //             {
-    //                 src: 'maskable-icon-512x512.png',
-    //                 sizes: '512x512',
-    //                 type: 'image/png',
-    //                 purpose: 'maskable'
-    //             }
-    //         ]
-    //     },
-    //     registerWebManifestInRouteRules: true,
-    //     workbox: {
-    //         navigateFallback: undefined,
-    //         cleanupOutdatedCaches: true,
-    //         globPatterns: ['**/*.{json,ico,svg,ttf,woff,css,scss,js,html,txt,jpg,png,woff2,mjs,otf,ani}'],
-    //         runtimeCaching: [
-    //             {
-    //                 urlPattern: "/",
-    //                 handler: 'NetworkFirst',
-    //             },
-    //             // {
-    //             //     urlPattern: /^https:\/\/api\.mapbox\.com\/.*/i,
-    //             //     handler: "CacheFirst",
-    //             //     options: {
-    //             //         cacheName: "mapbox-cache",
-    //             //         cacheableResponse: {
-    //             //             statuses: [0, 200],
-    //             //         },
-    //             //     },
-    //             // },
-    //         ],
-    //     },
-    //     client: {
-    //         installPrompt: false,
-    //         periodicSyncForUpdates: 20, //seconds
-    //     },
-    //     devOptions: {
-    //         enabled: true,
-    //         suppressWarnings: false,
-    //         navigateFallback: 'index.html',
-    //         type: 'module',
-    //     },
-    // },
+    pwa: {
+        registerType: 'autoUpdate',
+        manifest: {
+            name: 'Pico',
+            short_name: 'Pico',
+            theme_color: '#1867c0',
+            background_color: '#1867c0',
```

Comment:
I took these colors from commented-out config. Would appreciate suggestions for more appropriate ones.

Reply:
You can set them to white. Will need to make them dyanimic at runtime when dark theme is finished.